akka-http-core/src/main/scala/akka/http/impl/settings/ClientConnectionSettingsImpl.scala

```diff
@@ -24,7 +24,8 @@ private[akka] final case class ClientConnectionSettingsImpl(
   requestHeaderSizeHint: Int,
   websocketRandomFactory: () ⇒ Random,
   socketOptions: immutable.Seq[SocketOption],
-  parserSettings: ParserSettings)
+  parserSettings: ParserSettings,
+  redirectSettings: akka.http.scaladsl.settings.ClientAutoRedirectSettings)
```

Comment:
Thanks, it's good to know! Then how to mark these issues as ignored? 

Besides those I only have 2 errors in binary compatibility:
```
[error]  * abstract method getRedirectSettings()akka.http.javadsl.settings.ClientAut
oRedirectSettings in class akka.http.javadsl.settings.ClientConnectionSettings is pr
esent only in current version
```
and
```
[error]  * abstract method redirectSettings()akka.http.scaladsl.settings.ClientAutoR
edirectSettings in class akka.http.scaladsl.settings.ClientConnectionSettings is pre
sent only in current version
```
These are the newly added methods. Shouldn't they also be ignored for binary compatibility purposes? I mean, adding a method shouldn't impact the binary compatibility, should it?